dataactcore/read_config.py

```diff
@@ -4,7 +4,7 @@
 import yaml
 import re
 
-VALUES_TO_LOG = {"CONFIG_BROKER":["path","use_aws","local",'aws_bucket', 'aws_role', 'aws_region', 'aws_create_temp_credentials',
+"""VALUES_TO_LOG = {"CONFIG_BROKER":["path","use_aws","local",'aws_bucket', 'aws_role', 'aws_region', 'aws_create_temp_credentials',
```

Comment:
Thanks for incorporating @klrBAH's suggestion re: putting this info into the actual log file. Can this string come out of `read_config` now?